corehq/apps/api/tests/test_es.py

```diff
@@ -18,16 +18,11 @@ class TestESView(SimpleTestCase):
     def setUp(self):
         super().setUp()
         self.cases = case_adapter
-        self._purge_indices()
         manager.index_create(self.cases.index_name)
-        manager.index_put_mapping(self.cases.index_name, self.cases.type,
-                                       self.cases.mapping)
+        self.addCleanup(self._purge_indices())
```

Comment:
I think this should pass the function, not its result.
```suggestion
        self.addCleanup(self._purge_indices)
```

Reply:
Yep 🤦 
Thanks for catching that 😄